openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -439,10 +439,35 @@ class c02900(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "'Above the line' AGI deductions"
+    unit = USD
     documentation = (
         """Total of all 'above the line' income adjustments to get AGI"""
     )
 
+    def formula(tax_unit, period, parameters):
+        misc_haircuts = parameters(period).tax.ald.misc
+        return sum(
+            [
+                (1 - haircut) * tax_unit(variable, period)
+                for haircut, variable in (
+                    (misc_haircuts.c03260, "c03260"),
+                    (misc_haircuts.care_deduction, "care_deduction"),
+                    (misc_haircuts.student_loan, "filer_e03210"),
+                    (misc_haircuts.early_withdrawal, "filer_e03400"),
```

Comment:
Sorry to keep this PR going but could you align the haircut parameter name to the variable? e.g., `misc_haircuts.e03400` here. Would that then enable something simpler like:
```python
HAIRCUT_VARS = ["e03210", "e03400", ...]
return sum([(1 - misc_haircuts[variable]) * tax_unit(var, period) for variable in HAIRCUT_VARS])
```

Reply:
OK, how does it look now?